packages/eslint-plugin/docs/rules/indent.md

```diff
@@ -32,3 +34,13 @@ Taken with ❤️ [from ESLint core](https://github.com/eslint/eslint/blob/main/
 - [ ] ✅ Recommended
 - [x] 🔧 Fixable
 - [ ] 💭 Requires type information
+
+### Source
+
+<!-- markdownlint-disable MD044 -->
+
+- Rule: [indent.ts](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/rules/indent.ts)
+- Documentation: [indent.md](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/indent.md)
+- Tests: [indent.test.ts](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/tests/rules/indent/indent.test.ts) and [indent-eslint.test.ts](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/tests/rules/indent/indent-eslint.test.ts)
```

Comment:
I think it'd be easier & more consistent to use a single link to the tests folder instead of a link to each of the tests. Less code logic to maintain and such.